UnitTests/PaymentsCoreTests/APIClientTests.swift

```diff
@@ -0,0 +1,50 @@
+import XCTest
+@testable import PaymentsCore
+
+class APIClientTests: XCTestCase {
+    
+    lazy var apiClient: APIClient = {
+        APIClient(
+            urlSession: URLSession(urlProtocol: URLProtocolMock.self),
+            environment: .sandbox
+        )
+    }()
+
+    func testAccessTokenMockRequest_Success() {
```

Comment:
Nitpick, but I think this can help us establish good test naming from the start. Can we align on a naming convention for tests?

Something like `test<Subject/MethodUnderTestName>_when<Condition>_<ExpectedResult>`

So something like this in these tests:
`testFetch_withValidAccessTokenRequest_returnsValidAccessToken`
`testFetch_withInvalidAccessTokenRequest_returnsError`

I think this will help as the SDK gains a ton of tests.

Reply:
can't speak for Jon and Jax, but I like this a lot!!! :smile:

Reply:
I think this sounds really good. Will update!

Reply:
+1 to this as well, I think that sounds good!

Reply:
updated!